Library/Formula/openvdb.rb

```diff
@@ -0,0 +1,100 @@
+class Openvdb < Formula
+  desc "Sparse volume processing toolkit"
+  homepage "http://www.openvdb.org/"
+  url "https://github.com/dreamworksanimation/openvdb/archive/v3.0.0.tar.gz"
+  sha256 "6c90cfda032c54876b321031717c13ea56a6b7b15c911d3edfbb2ad7af49700e"
+  head "https://github.com/dreamworksanimation/openvdb.git"
+
+  bottle do
+    sha256 "843a6b55c84185101c1e1b379751e99cd0215f2fce7534ef3a2387a33a7c91a5" => :yosemite
+  end
+
+  option 'tests', 'Installs and runs the unit tests for the OpenVDB library'
+  option 'logging', 'Requires log4cplus'
+  option 'docs', 'Installs documentation'
+
+  depends_on "boost"
+  depends_on "openexr"
+  depends_on "ilmbase"
+  depends_on "tbb"
+  depends_on "jemalloc" => :recommended
+  depends_on "homebrew/versions/glfw3" => :recommended
```

Comment:
Can't have mandatory or recommended deps from taps in core formulae. This will need to be `optional`.